src/app/space/create/codebases/codebases-item/codebases-item.component.ts

```diff
@@ -58,7 +60,11 @@ export class CodebasesItemComponent implements OnDestroy, OnInit {
    */
   getNotificationMessage(): string {
     if (this.cheState) {
-      return this.cheState.running ? this.cheRunningMessage : this.cheStartingMessage;
+      if (this.cheState.multiTenant) {
+        return this.cheState.running ? this.cheFinishedMultiTenantMigrationMessage : this.chePerformingMultiTenantMigrationMessage;
+      } else {
+        return this.cheState.running ? this.cheRunningMessage : this.cheStartingMessage;
+      }
```

Comment:
The code looks ok to me, but what happens when Che is idle? That is, when does the migration kick in?

Reply:
if che is idled it is not a problem for migration - migration pod will start single-tenant che-server, migrate workspaces to multi-tenant one, and than remove single-tenant deployment